spec.html

```diff
@@ -9,15 +10,16 @@
 contributors: Guy Bedford, Bradley Farias, Michael Ficarra
 </pre>
 
-<emu-clause id="sec-demo-clause">
-  <h1>This is an emu-clause</h1>
-  <p>This is an algorithm:</p>
-  <emu-alg>
-    1. Let _proposal_ be *undefined*.
-    1. If _proposal_ is ~accepted~, then
-      1. Let _stage_ be 0.
-    1. Else,
-      1. Let _stage_ be -1.
-    1. Return ? ToString(_proposal_).
-  </emu-alg>
+<emu-clause number="22.1.3" id="sec-properties-of-the-string-prototype-object">
```

Comment:
Does this work? I'd expect "number" to only govern within the section, meaning you'd need this:

```suggestion
<emu-clause number="22">
  <h1>Text Processing</h1>

  <emu-clause number="1">
    <h1>String Objects</h1>

    <emu-clause number="3" id="sec-properties-of-the-string-prototype-object">
```

Reply:
It does work. See https://tc39.es/ecmarkup/#emu-clause

Reply:
very interesting, thanks